4.LastFM/lastfmLoader.js

```diff
@@ -0,0 +1,21 @@
+function LastfmLoader(key, url) {
+  this.key = key || 'ea9abab35aef9b1a492f071c30b761b2';
+  this.url = url || 'http://ws.audioscrobbler.com/2.0/?format=json&';
+  this.query = '';
+  this.load = function(callback) {
+    var URL = this.url + 'api_key=' + this.key + '&' + this.query;
```

Comment:
I mean url is `http://ws.audioscrobbler.com/2.0/` and query params are `api_key`, `author`, `format` etc